### DISCUSSION TITLE
Release v3.0.0

### DISCUSSION THREAD

**Discussion by PacificGilly** [farridav/django-jazzmin#559] [General]
Hey @farridav (and everyone else), I just want to discuss whether we're happy to release v3.0.0 of `django-jazzmin` now since we have a couple of fixes users have requested (the logout issue—https://github.com/farridav/django-jazzmin/pull/544) and have added/removed support for various Python versions.

https://github.com/farridav/django-jazzmin/releases/tag/untagged-21539373cb46f66d3e8f

**Comment by JMSoler7** [comment]
Considering that fixing issue #544 is essential, I also encourage releasing a new version that includes the commit that addresses it!
